api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "unversioned"
+   "version": "v1.6.0"
```

Comment:
also about the cherry-pick, yes, we need to regenerate spec on cherry pick. but we probably need to do that anyway if there is any type-changed between master and release branch (though the auto-merge may take care of most of them)